fe/fe-core/src/main/java/com/starrocks/catalog/CatalogUtils.java

```diff
@@ -352,4 +353,22 @@ public static int calAvgBucketNumOfRecentPartitions(OlapTable olapTable, int rec
         return bucketNum;
     }
 
+    public static String addEscapeCharacter(String comment) {
+        if (StringUtils.isEmpty(comment)) {
+            return comment;
+        }
+        StringBuilder output = new StringBuilder();
+        for (int i = 0; i < comment.length(); i++) {
```

Comment:
only support \\ \n ? if sql is constructed by a program, it may be contains other escape characters,  if we do not support other escaped characters, reporting error when encountering them is  just OK

Reply:
Most `escape characters` processed in `escapeBackSlash()` not affect the string meaning except '\' and '"'`, so special process for these two characters.